lang/py/src/avro/schema.py

```diff
@@ -312,6 +312,18 @@ def name_ref(self, names):
   namespace = property(lambda self: self.get_prop('namespace'))
   fullname = property(lambda self: self._fullname)
 
+# Logical type class
+class LogicalSchema(Schema):
```

Comment:
You might want to base on https://github.com/Yelp/avro/blob/trunk/lang/py/src/avro/schema.py#L322.